scipy/signal/__init__.py

```diff
@@ -379,7 +386,7 @@ def wrapped(*args, **kwargs):
 
 del deprecated_windows, name, deco
 
-__all__ = [s for s in dir() if not s.startswith('_')]
+__all__ = [s for s in dir() if not s.startswith('_') and s!="warnings"]
```

Comment:
To be added with `[skip ci]` before merging
```suggestion
__all__ = [s for s in dir() if not s.startswith('_') and s !="warnings"]
```

Reply:
I saw this too, but though it's not important enough to note. However, if you do fix it, please add a space on both sides of the `!=`.